EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigiGPUKernel.cuh

```diff
@@ -31,11 +27,12 @@ const uint32_t ROC_bits         = 5;
 const uint32_t DCOL_bits        = 5;
 const uint32_t PXID_bits        = 8;
 const uint32_t ADC_bits         = 8;
+
 // special for layer 1
-const uint32_t LINK_bits1       = 6;
-const uint32_t ROC_bits1        = 5;
-const uint32_t COL_bits1_l1     = 6;
-const uint32_t ROW_bits1_l1     = 7;
+const uint32_t LINK_bits_l1     = 6;
+const uint32_t ROC_bits_l1      = 5;
+const uint32_t COL_bits_l1      = 6;
+const uint32_t ROW_bits_l1      = 7;
```

Comment:
Should these be in a namespace to not pollute the global namespace with them?

(the question is more for the existing code than this PR though)

Reply:
Yes.

Also, they are possibly duplicated somewhere else.

Reply:
Suggestions for the `namespace` name ?